plugins/frontend/backstage-plugin-github-pull-requests/src/components/Home/RequestedReviewsCard/Content.test.tsx

```diff
@@ -77,4 +77,22 @@ describe('<RequestedReviewsCard>', () => {
       await screen.findByText("Test PR don't merge", { exact: false }),
     ).toBeInTheDocument();
   });
+  it('should render home card with requested reviews, using a custom query', async () => {
+    const customQuery = "is:open is:pr review-requested:@me archived:false is:draft"
+    render(
+      wrapInTestApp(
+        <TestApiProvider apis={apis}>
+          <Content query={customQuery} />
+        </TestApiProvider>,
+      ),
+      {},
+    );
+
+    expect(
+      await screen.findByText('Revert "Sc 7454 AWS S3 docs (#640)"', { exact: false }),
+    ).toBeInTheDocument();
+    expect(
+      screen.queryByText("Test PR don't merge", { exact: false }),
```

Comment:
FYI that I use `queryByText` here as I'm checking for the absence of something and `findByText` will throw an error before reaching the `.not.toBeInTheDocument()` assertion.

`queryByText` returns null instead which satisfies the check that the query has been applied (by way of the returned mock data being different)